styles/utils.css

```diff
@@ -17,5 +17,98 @@
     --primary-button-hover: rgb(21, 50, 180);
     --secondary-button-hover: rgb(236, 233, 233);
     --secondary-hover: rgb(220, 219, 245);
```

Comment:
either or hex code or name or RGB. don't mix all of them

Reply:
Noted, will update at the end else there will be merge conflicts.